src/observable/ArrayLikeObservable.ts

```diff
@@ -55,7 +55,8 @@ export class ArrayLikeObservable<T> extends Observable<T> {
 
   protected _subscribe(subscriber: Subscriber<T>): TeardownLogic {
     let index = 0;
-    const { arrayLike, scheduler } = this;
+    const arrayLike = this.arrayLike;
+    const scheduler = this.scheduler;
```

Comment:
Any reason? I think we want to keep our existing, destructure style. Is there a subtle difference I'm missing?